app/src/main/java/fr/gaulupeau/apps/Poche/data/WallabagDbOpenHelper.java

```diff
@@ -51,6 +53,21 @@ public void onUpgrade(Database db, int oldVersion, int newVersion) {
                     }
                 }
 
+                if (oldVersion < 103) {
+                    Log.i(TAG, "Migrating to version " + 103);
+
+                    ArticleContentDao.createTable(db, false);
+
+                    db.execSQL("insert into " + ArticleContentDao.TABLENAME +
+                            "(" + ArticleContentDao.Properties.Id.columnName +
+                            ", " + ArticleContentDao.Properties.Content.columnName + ")" +
+                            " select " + ArticleDao.Properties.Id.columnName + ", CONTENT" +
+                            " from " + ArticleDao.TABLENAME);
+
+                    // SQLite can't drop columns; just removing the data
```

Comment:
we could only work around this, if we copy the data to a new table, drop this table and rename the new table to the old name. (e.g. https://stackoverflow.com/a/5987838/709697)
The question is, do we want to do this?

Reply:
Nah, not worth the trouble.
Greendao doesn't provide any migrations, so its our job. And it doesn't care on this early stages of loading.